lib/fizzy/execute.cpp

```diff
@@ -396,6 +396,55 @@ execution_result execute(Instance& instance, FuncIdx func_idx, std::vector<uint6
             labels.push_back(label);
             break;
         }
+        case Instr::if_:
+        {
+            if (static_cast<uint32_t>(stack.pop()) != 0)
+            {
+                const auto arity = read<uint8_t>(immediates);
+                const auto target_pc = read<uint32_t>(immediates);
+                const auto target_imm = read<uint32_t>(immediates);
```

Comment:
The first 3 fields are shared. @gumb0 @chfast Would it make sense moving them above the if?

Reply:
Yep.